src/backend/database.json

```diff
@@ -11,7 +11,11 @@
         "9": "We don't do your homework, so use your brain and the help provided. We'll point you in the right direction most of the time.",
         "10": "The CPPreference bot is used like this `.cppref strcpy` where `strcpy` is what you need to find, it will retrieve the link of the cppreference website and post it in the channel.",
         "11": "Don't ping us unless there is an issue that needs an admin to be solved. (If you do, I'll warn you, then ban you).",
-        "12": "Ask people before going into private messages."
+        "12": "Ask people before going into private messages.",
+        "13": "Any question / discussion about hacking is not allowed, it will get you instantly banned.",
+        "14": "If any staff / admin tells you to stop and you do not stop you will receive punishment.",
+        "15": "I shouldn't have to say that but any NSFW stuff will be deleted, and you'll probably get banned alongside it. Don't post NSFW, this includes sex talk.",
+        "16": "POSTING ADVERTISEMENT OF ANY KIND WILL GET YOU BANNED. (Except in #show-off-your-stuff  for obvious reasons)"
```

Comment:
There's a extra space between `#show-off-your-stuff` and `for`. Otherwise seems good to me.